deploy/osd-14634-disable-cpms/01-osd-disable-cpms.Role.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: osd-disable-cpms
+  namespace: openshift-machine-api
+rules:
+# Service account needs to be able to get pods and exec on them
+- apiGroups:
+  - machine.openshift.io
+  resources:
+  - controlplanemachinesets
+  verbs:
```

Comment:
Are all these perms needed? It looks like we might just need delete.

Reply:
Good call!